apim/README.md

```diff
@@ -25,7 +25,7 @@ $ helm package .
 To install the chart using the chart archive, run:
 
 ```
-$ helm install gravitee-1.23.2.tgz
+$ helm install apim-1.27.1.tgz
```

Comment:
did you think about auto generating them and hosting them on github?

Reply:
you can find some example here: https://github.com/orchestracities/charts

Reply:
> did you think about auto generating them and hosting them on github?

Yes, I'm currently working on this but I have some ssl certificates issues with github pages when adding the helm repo:

```shell
helm repo add graviteeio https://helm.gravitee.io
Error: Looks like "https://helm.gravitee.io" is not a valid chart repository or cannot be reached: Get https://helm.gravitee.io/index.yaml: x509: certificate is valid for www.github.com, *.github.io, *.githubusercontent.com, *.github.com, github.com, github.io, githubusercontent.com, not helm.gravitee.io
```

Reply:
I think it works well for you because you're still using a github domain

> helm repo index . --url https://orchestracities.github.io/charts/

Reply:
right, not sure you can do that if not with a redirect . not sure this can help: https://github.blog/2018-05-01-github-pages-custom-domains-https/

Reply:
I already had a look with no luck...

Reply:
Don't know what was the issue with DNS records but seems to be ok now (maybe due to dns propagation?)